linter_requirements.txt

```diff
@@ -1,2 +1,3 @@
 pycodestyle==2.8.0
 GitPython>=3.1.30
+spin==0.7
```

Comment:
Ci failure seems real, I guess the `spin lint` is trying to rebuild so it needs a pretty full dev env?
Also, should this be `>=`?